Logos/intel471_logo.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 338.93 104.05"><defs><style>.cls-1{fill:#d6002f;}.cls-2{fill:#121212;}</style></defs><g id="Layer_2" data-name="Layer 2"><g id="Layer_2-2" data-name="Layer 2"><path class="cls-1" d="M49.72,104.05v-60h6.85v60Z"/><path class="cls-1" d="M112.26,44.06v60h-6.88v-.23L75.47,55v49.05H68.63v-60H76.4l29,47.44V44.06Z"/><path class="cls-1" d="M157.85,44.13V51H141.39v53h-6.84V51H118.1V44.13h16.45v0h6.84v0Z"/><path class="cls-1" d="M196.7,97.22v6.83H168.34v0h-3.42V44.24h6.84v.08h24.86v6.84H171.76v19.9h19.32v6.83H171.76V97.22Z"/><path class="cls-1" d="M236.4,97.19V104H204v-60h6.86v53.1Z"/><path class="cls-2" d="M267.48,44.22H256.94L234.07,79.43l-.28.42v9.36H257.5v14.84h10V89.2h4.69V79.26h-4.69ZM246,79.25,257.5,61.68V79.25Z"/><path class="cls-2" d="M313.81,44.31H274.07V54.43h22a58.57,58.57,0,0,0-7.62,8.89,68.07,68.07,0,0,0-9.62,20.36,77.09,77.09,0,0,0-2.79,18.59l-.06,1.78h10.17l0-1.69a59.51,59.51,0,0,1,2.31-16.06,55.3,55.3,0,0,1,8.22-16.76A56.88,56.88,0,0,1,313,54.17l.81-.51Z"/><polygon class="cls-2" points="329.57 44.09 313.62 59.83 320.56 66.86 329.04 58.5 329.04 104.05 338.93 104.05 338.93 44.09 329.57 44.09"/><path class="cls-1" d="M37.21,34.81,31.69,31s-.76,2-3.73,2c-2.71,0-4.58-3.59-2.58-6.26l-4-2.74a9,9,0,0,0-2.64,6.27c0,5.1,5.1,8.1,9.24,8.1C34.22,38.33,37.21,34.81,37.21,34.81Z"/><path class="cls-1" d="M42.81,34.81,48.33,31s.75,2,3.72,2c2.72,0,4.59-3.59,2.59-6.26l4-2.74a9,9,0,0,1,2.65,6.27c0,5.1-5.11,8.1-9.25,8.1C45.79,38.33,42.81,34.81,42.81,34.81Z"/><polygon class="cls-2" points="48.66 6 31.69 6 39.98 12.94 48.66 6"/><path class="cls-2" d="M63.75,13.56a11.21,11.21,0,0,0,3.46-8A10.52,10.52,0,0,0,65.75,0L40,22.84,14.2,0a10.47,10.47,0,0,0-1.47,5.54A10.67,10.67,0,0,0,16,13.33S0,28,0,53.54C0,94,19.39,103.93,19.39,103.93s-10.74-18-7.67-35.19c1-5.71,3.67-9.41,7-12.9A28.17,28.17,0,0,1,35.18,47.6L40,56V40A49.25,49.25,0,0,0,22.1,44.68,33.18,33.18,0,0,0,9.22,56.06,44.49,44.49,0,0,1,12.6,33.65c4.27-10.14,9.88-15.2,9.88-15.2L40,31.29,56.81,18.5S64.22,22.78,68,33.75c.65,1.92,1.63,4.58,1.63,4.58C72.68,25.62,63.75,13.56,63.75,13.56Z"/></g></g></svg>
```

Comment:
@mmolenda, do not use illegal attributes: data-name, style, xmlns:link. And please use unique guid for id value, thanks.